### PR TITLE
Metrics fix: replace the scheduler_binding_latency_microseconds with …

### PR DESCRIPTION
**Pull request by perk-sumo**
…scheduler_binding_duration_seconds

###### Description

Fill in your description here.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/prometheus-overrides.yaml [hunk 1]
```diff
@@ -207,7 +207,7 @@ prometheus:
       # scheduler_e2e_scheduling_latency_microseconds
```
Comment (justrelax19) [inline deploy/helm/prometheus-overrides.yaml:207]: @perk-sumo I think these metrics also got changed from `scheduler_e2e_scheduling_latency_microseconds` to `scheduler_e2e_scheduling_latency_seconds`

File: deploy/helm/sumologic/values.yaml [hunk 2]
```diff
@@ -954,7 +954,7 @@ prometheus-operator:
         # scheduler_e2e_scheduling_latency_microseconds
```
Comment (justrelax19) [inline deploy/helm/sumologic/values.yaml:954]: Same as above

File: deploy/helm/prometheus-overrides.yaml (hunk 1 above)
Comment (perk-sumo) [reply deploy/helm/prometheus-overrides.yaml:207]: Actually it changed to `e2e_scheduling_duration_seconds`
https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/metrics/metrics.go#L74

File: deploy/helm/sumologic/values.yaml [hunk 3]
```diff
@@ -951,19 +951,19 @@ prometheus-operator:
...
-        # scheduler_scheduling_algorithm_latency_microseconds
-        # scheduler_scheduling_algorithm_latency_microseconds_bucket
-        # scheduler_scheduling_algorithm_latency_microseconds_count
+        # scheduler_e2e_scheduling_duration_seconds
+        # scheduler_e2e_scheduling_duration_seconds_bucket
+        # scheduler_e2e_scheduling_duration_seconds_count
+        # scheduler_binding_duration_seconds
+        # scheduler_binding_duration_seconds_bucket
+        # scheduler_binding_duration_seconds_count
+        # scheduler_scheduling_algorithm_duration_seconds
```
Comment (perk-sumo) [inline deploy/helm/sumologic/values.yaml:960]: https://github.com/kubernetes/kubernetes/blob/release-1.16/pkg/scheduler/metrics/metrics.go#L116

File: deploy/helm/sumologic/values.yaml [hunk 4]
```diff
@@ -951,19 +951,19 @@ prometheus-operator:
...
-        # scheduler_binding_latency_microseconds
-        # scheduler_binding_latency_microseconds_bucket
-        # scheduler_binding_latency_microseconds_count
-        # scheduler_scheduling_algorithm_latency_microseconds
-        # scheduler_scheduling_algorithm_latency_microseconds_bucket
-        # scheduler_scheduling_algorithm_latency_microseconds_count
+        # scheduler_e2e_scheduling_duration_seconds
+        # scheduler_e2e_scheduling_duration_seconds_bucket
+        # scheduler_e2e_scheduling_duration_seconds_count
+        # scheduler_binding_duration_seconds
```
Comment (perk-sumo) [inline deploy/helm/sumologic/values.yaml:957]: https://github.com/kubernetes/kubernetes/blob/release-1.16/pkg/scheduler/metrics/metrics.go#L188

File: deploy/helm/sumologic/values.yaml [hunk 5]
```diff
@@ -951,19 +951,19 @@ prometheus-operator:
...
-        # scheduler_e2e_scheduling_latency_microseconds
-        # scheduler_e2e_scheduling_latency_microseconds_bucket
-        # scheduler_e2e_scheduling_latency_microseconds_count
-        # scheduler_binding_latency_microseconds
-        # scheduler_binding_latency_microseconds_bucket
-        # scheduler_binding_latency_microseconds_count
-        # scheduler_scheduling_algorithm_latency_microseconds
-        # scheduler_scheduling_algorithm_latency_microseconds_bucket
-        # scheduler_scheduling_algorithm_latency_microseconds_count
+        # scheduler_e2e_scheduling_duration_seconds
```
Comment (perk-sumo) [inline deploy/helm/sumologic/values.yaml:954]: https://github.com/kubernetes/kubernetes/blob/release-1.16/pkg/scheduler/metrics/metrics.go#L98

**Comment by perk-sumo** [comment]
I confirmed those metrics are available in `1.16`, we are good to go with the PR
https://github.com/kubernetes/kubernetes/blob/release-1.16/pkg/scheduler/metrics/metrics.go

**Pull request merged**